### ISSUE TITLE
CoordinateSystem.WKT generates WKT that CoordinateSystemWktReader cannot parse

### ISSUE THREAD

**Issue by jarobbins** [NetTopologySuite/ProjNet4GeoAPI#76]
The following is a sample WKT I'm reading from an external source:
```cs
string sampleWKT =
                "PROJCS[\"\", " +
                  "GEOGCS[\"\", " +
                    "DATUM[\"\", " +
                      "SPHEROID[\"GRS_1980\", 6378137, 298.2572221010042] " +
                    "], " +
                  "PRIMEM[\"Greenwich\", 0], " +
                  "UNIT[\"Degree\", 0.017453292519943295]" +
                  "], " +
                  "PROJECTION[\"Transverse_Mercator\"], " +
                  "PARAMETER[\"False_Easting\", 500000], " +
                  "PARAMETER[\"False_Northing\", 0], " +
                  "PARAMETER[\"Central_Meridian\", -75], " +
                  "PARAMETER[\"Scale_Factor\", 0.9996], " +
                  "UNIT[\"Meter\", 1]" +
                  "]";
```

If I parse that sample WKT, everything is fine:
```cs
var csFromSample = (CoordinateSystem)ProjNet.IO.CoordinateSystems.CoordinateSystemWktReader.Parse(sampleWKT);  // does not throw
```

If I save the WKT from `csFromSample` and attempt to re-parse it later, I get an exception from `CoordinateSystemWktReader.ReadProjection`:
```cs
string wktFromProjNetCS = csFromSample.WKT;
var csFromProjNetGeneratedWKT = (CoordinateSystem)ProjNet.IO.CoordinateSystems.CoordinateSystemWktReader.Parse(wktFromProjNetCS);  // throws
``` 

The parser error is the same one mentioned in #65.  To save some space, the only difference between the original sample WKT and the value stored in `wktFromProjNetCS` is that the `UNIT` entry comes before the `PROJECTION` entry, instead of after the last `PARAMETER` entry.  If I put any other keys after the last `PARAMETER` entry (ex. `AXIS`, `UNIT`, etc.), it will parse without issue.

What I don't know is which version of the WKT is "most correct".  Must `UNIT` come after the `PARAMETER` entries?  Or, should the `CoordinateSystemWktReader.ReadProjection` method be updated to allow `UNIT` to be listed as the `CoordinateSystem.WKT` property returns it?

**Comment by jarobbins** [comment]
Add failing test case in PR #77.

**Comment by jarobbins** [comment]
For clarity, here is the WKT generated by CoordinateSystem.WKT (the WKT that will not parse correctly):

PROJCS["",
	   GEOGCS["",
			  DATUM["",
					SPHEROID["GRS_1980", 6378137, 298.2572221010042]
					],
			  PRIMEM["Greenwich", 0],
			  UNIT["Degree", 0.017453292519943295]
			  ],
	   UNIT["Meter", 1],
	   PROJECTION["Transverse_Mercator"],
	   PARAMETER["False_Easting", 500000],
	   PARAMETER["False_Northing", 0],
	   PARAMETER["Central_Meridian", -75],
	   PARAMETER["Scale_Factor", 0.9996]
	   ]

**Comment by airbreather** [comment]
Closed by #77.
